doc/nrf/known_issues.rst

```diff
@@ -1503,7 +1504,7 @@ DRGN-16317: The SoftDevice Controller may discard LE Extended Advertising Report
 DRGN-16341: The SoftDevice Controller may discard LE Extended Advertising Reports
   Extended Advertising Reports with data length of 228 are discarded.
 
-.. rst-class:: v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
+.. rst-class:: v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
```

Comment:
This should not have NCS 1.8.0 and NCS 1.9.0?
https://github.com/nrfconnect/sdk-nrf/pull/5784
@rugeGerritsen

Reply:
@thomasstenersen , the following line is correct. The affected versions are placed above the known issue text. So this refers to DRGN-15903 which is present in both NCS 1.8 and NCS 1.9